multinode_integration_tests/tests/verify_bill_payment_utils/utils.rs

```diff
@@ -0,0 +1,1003 @@
+use node_lib::accountant::db_access_objects::receivable_dao::{ReceivableDao, ReceivableDaoReal};
+use node_lib::accountant::gwei_to_wei;
+use node_lib::blockchain::bip32::Bip32EncryptionKeyProvider;
+use node_lib::blockchain::blockchain_interface::blockchain_interface_web3::{
+    BlockchainInterfaceWeb3, REQUESTS_IN_PARALLEL,
+};
+use node_lib::blockchain::blockchain_interface::lower_level_interface::{
+    LowBlockchainInt, ResultForBalance,
+};
+use node_lib::blockchain::blockchain_interface::BlockchainInterface;
+use node_lib::database::db_initializer::{
+    DbInitializationConfig, DbInitializer, DbInitializerReal, ExternalData,
+};
+use node_lib::sub_lib::accountant::PaymentThresholds;
+use node_lib::sub_lib::blockchain_interface_web3::transaction_data_web3;
+use node_lib::sub_lib::wallet::Wallet;
+use node_lib::test_utils;
+use node_lib::test_utils::standard_dir_for_test_input_data;
+use rustc_hex::{FromHex, ToHex};
+use std::cell::RefCell;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+use std::thread;
+use std::time::{Duration, Instant, SystemTime};
+use tiny_hderive::bip32::ExtendedPrivKey;
+use web3::transports::Http;
+use web3::types::{Address, Bytes, SignedTransaction, TransactionParameters, TransactionRequest};
+use web3::Web3;
+
+pub type StimulateConsumingNodePayments = fn(&mut MASQNodeCluster, &MASQRealNode, &WholesomeConfig);
+
+pub type StartServingNodesAndLetThemPerformReceivablesCheck =
+    fn(&mut MASQNodeCluster, &WholesomeConfig) -> [MASQRealNode; 3];
+
+pub fn test_body(
+    test_inputs: TestInputs,
+    assertions_values: AssertionsValues,
+    stimulate_consuming_node_to_pay: StimulateConsumingNodePayments,
+    start_serving_nodes_and_activate_their_accountancy: StartServingNodesAndLetThemPerformReceivablesCheck,
+) {
+    // It's important to prevent the blockchain server handle being dropped too early
+    let (mut cluster, global_values, _blockchain_server) = establish_test_frame(test_inputs);
+    let consuming_node =
+        global_values.prepare_consuming_node(&mut cluster, &global_values.blockchain_interfaces);
+    let serving_nodes_array = global_values.prepare_serving_nodes(&mut cluster);
+    global_values.set_up_consuming_node_db(&serving_nodes_array, &consuming_node);
+    global_values.set_up_serving_nodes_databases(&serving_nodes_array, &consuming_node);
+    let wholesome_config = WholesomeConfig::new(global_values, consuming_node, serving_nodes_array);
+    wholesome_config.assert_expected_wallet_addresses();
+    let real_consuming_node = cluster.start_named_real_node(
+        &wholesome_config
+            .consuming_node
+            .common
+            .prepared_node
+            .node_docker_name,
+        wholesome_config.consuming_node.common.prepared_node.index,
+        wholesome_config
+            .consuming_node
+            .common
+            .startup_config_opt
+            .borrow_mut()
+            .take()
+            .unwrap(),
+    );
```

Comment:
It was noticed during the review that `wholesome_config.consuming_node.common` is common in all the three arguments. I believe (and so do you) it is justified if we create a new variable called `consuming_node_common`.